### PR TITLE
DEN-101: Get symfony 5 compatible / upgrade php

### PR DESCRIPTION
**Pull request by migo315**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,20 @@ bestit_commercetools_odm:
...
+        factory: 'BestIt\CommercetoolsODMBundle\Factory\ClientFactory:create'
+        arguments:
+            - {client_id: '%env(COMMERCETOOLS_CLIENT_ID)%', client_secret: '%env(COMMERCETOOLS_CLIENT_SECRET)%', project: '%env(COMMERCETOOLS_PROJECT)%', scope: ['manage_project'] }
+            - {locale: 'de', languages: ['de'] }
+
+    BestIt\CommercetoolsODMBundle\Factory\ClientFactory:
+        class: BestIt\CommercetoolsODMBundle\Factory\ClientFactory
+        arguments: ['@cache.app', '@logger']
+```
+
```
Comment (migo315) [inline README.md:74]: We often copied this factory from project to project. So I put them in the ODM.

**Review by avareldev — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -11,23 +11,18 @@
...
-    },
     "type": "lib",
     "require": {
-        "php": "^7.0",
-        "bestit/commercetools-odm": "^1.0@dev",
-        "symfony/dependency-injection": "^3.1 || ^4.0",
-        "symfony/yaml": "^3.1 || ^4.0",
-        "symfony/http-kernel": "^3.1 || ^4.0"
+        "php": "^7.2.5",
+        "bestit/commercetools-odm": "^0.55.3",
```
Comment (b3nl) [inline composer.json]: What happened here?

**Review by b3nl — COMMENTED** [review]
Nice, but i am not sure with one point.

File: composer.json (hunk 2 above)
Comment (migo315) [reply composer.json]: Symfony 5 requires 7.2.5. That was the right time to update our requirements.

And the odm lib without a concrete version has often caused conflicts. Each library, which has a concrete odm version as dependency, conflicts with this bundle.

Comment (migo315) [reply composer.json]: Updated to odm 1.0

**Review by b3nl — APPROVED** [review]

**Pull request merged**